java-frontend/src/main/java/org/sonar/java/se/ExplodedGraphWalker.java

```diff
@@ -734,19 +734,24 @@ private static boolean isNonNullMethod(Symbol symbol) {
     return !symbol.isUnknown() && symbol.metadata().isAnnotatedWith("javax.annotation.Nonnull");
   }
 
+  /**
+   * We do nothing for local variables without initializers, for other variables we use default value or initializer if there is one
+   *
+   * See <a href="http://docs.oracle.com/javase/specs/jls/se8/html/jls-4.html#jls-4.12.5">JLS 4.12.5</a> for details
```

Comment:
really minor but in order to lighten the read we usually refer to JLS8 4.12.5 without the link...